generators/add-action/asset-compute/index.js

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// #########################################################################################
+// 3rd party template
+// #########################################################################################
+
+const path = require('path')
+const ActionGenerator = require('../../../lib/ActionGenerator')
+
+class AssetComputeGenerator extends ActionGenerator {
+  constructor (args, opts) {
+    super(args, opts)
+    this.props = {}
+  }
+
+  async prompting () {
+    this.props.actionName = await this.promptForActionName('example of a custom worker for the Adobe Asset Compute service', 'example')
```

Comment:
yesterday jesse commented out the line below so the action file is in the location aio expects (not in the root of the project).
```
// this.props.actionDestPath = this.destinationPath(`worker-${this.props.actionName}.js`)
```

This made the default action name: `example`. If we are removing that line and removing the `worker-` prefix from the worker file, we should change the default action name to something other than `example`, maybe just `worker`